scripts/run_tests.py

```diff
@@ -451,6 +451,43 @@
     "exit",
 ]
 
+cwl_cromwell_tests_list = [
+    "cwl_ad_hoc_file_test",
+    "cwl_cache_between_workflows",
+    "cwl_cache_within_workflow",
+    "cwl_docker_size",
+    "cwl_dynamic_initial_workdir",
+    "cwl_expressionLib",
+    "cwl_format",
+    "cwl_format_url",
+    "cwl_glob_sort",
+    "cwl_hello",
+    "cwl_http_inputs", # invalid input
```

Comment:
I think they are annotations on currently failing tests (reason why they are failing). Should clarify how this will be handled before merging -- I see on other tests that @jdidion 's approach is to comment out the whole line, and leave a comment on the right about what fix is pending to make the test work.

Reply:
To do: plan for fixing these on a case-by-case basis? They look like they are failing for a variety of reasons. E.g. "Could not resolve host: metadata.google.internal" is probably something we don't have access to.